.github/workflows/test.yml

```diff
@@ -0,0 +1,42 @@
+name: "CI"
+on:
+  pull_request:
+    paths-ignore:
+    - '**.md'
+    - '**.org'
+    - '.dir-locals.el'
+    branches:
+      - master
+  push:
+    paths-ignore:
+    - '**.md'
+    - '**.org'
+    - '.dir-locals.el'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        emacs_version:
+          - 24.5
+          - 25.3
+          - 26.3
+          - 27.1

```

Comment:
Shouldn't this also include `25.1`, `25.2`, `26.1`, and `26.2`?

Reply:
Sure